### PR TITLE
added UI UX case study for doubt solving website

### PR DESCRIPTION
**Pull request by ANSHIKA-26**

### CODE REVIEW

**Review by Nitya-Pasrija — COMMENTED** [review]
Perfection! Advancing through the open source world!🎊🎊 :) Please be patient while our maintainers and reviewers check the code.

**Review by Nitya-Pasrija — CHANGES_REQUESTED** [review]
The background in the design and the rating stars is not a clear UI, would recommend changing it

**Comment by Nitya-Pasrija** [comment]
@ANSHIKA-26 
Any updates?

File: Doubtstrom/Doubtstrom.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+To design a doubt solving platform for a univeristy , where  juniors can ask doubts from their seniors , making a ui ux case study on the same , the main point would be to ensure the credibility of the answer.
+
+## Solution
+A platform named DoubtStrom for the students , it had various features like selecting which year the student is from , this ensures that only students in same year / seniors can answer doubts of juniors .
+Futrther the rating of answers show the credibilty of the answers.
+
+
+Canva link for ux case study : https://www.canva.com/design/DAFXdpO5LZs/uLrIs3pOt2iOBewdePH6zg/edit?utm_content=DAFXdpO5LZs&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton
+
+Figma link for ui ux:https://www.figma.com/file/9Di7Or75R1Idb9IU6NjgYA/Doubtstrom?type=design&node-id=18%3A3&mode=design&t=GYGJIwhBw5G7IHnf-1
```
Comment (ANSHIKA-26) [inline Doubtstrom/Doubtstrom.md:11]: New figma link : https://www.figma.com/file/9Di7Or75R1Idb9IU6NjgYA/Doubtstrom?type=design&node-id=18%3A3&mode=design&t=sBHDCwsLBNtL9DKN-1

Comment (PriyaDeveloper630) [reply Doubtstrom/Doubtstrom.md:11]: > New figma link : https://www.figma.com/file/9Di7Or75R1Idb9IU6NjgYA/Doubtstrom?type=design&node-id=18%3A3&mode=design&t=sBHDCwsLBNtL9DKN-1

Would recommend adding more pages and functionalities to the design.

**Review by Nitya-Pasrija — APPROVED** [review]

**Pull request merged**